eduNEXT/tutor-contrib-edunext-distro
# feat: add nutmeg compatibility (DS-211)

This PR makes distro plugin able to work with nutmeg (nuez) version.

This release candidate only installs eox-tenant. 

**How to test**

- Use tutor  `>=14.0.5 ` to avoid errors related to edx-platform/pull/30569
- If you are using stack builder you can use a strain like and run `stack strain <local | dev > configure`
```yml
STRAIN_NAME: nuezdistro
STRAIN_TUTOR_VERSION: v14.0.5
CMS_HOST: studio.nutmeg.edunext.link
LMS_HOST: lms.nutmeg.edunext.link
DEV_PROJECT_NAME: nuezdistro_dev
LOCAL_PROJECT_NAME: nuezdistro_local
STRAIN_TUTOR_PLUGINS:
  - REPO: tutor-contrib-edunext-distro
    VERSION: dcoa/nuez-rc1 
    EGG: tutor-contrib-edunext-distro==v3.0.0
    PACKAGE_NAME: distro
    PROTOCOL: ssh
STRAIN_EXTRA_COMMANDS:
  - APP: 'echo'
    COMMAND: 'epa'
``` 
Or only install the plugin in your environment `pip install git+https://github.com/eduNEXT/tutor-contrib-edunext-distro@dcoa/nuez-rc1` and enable the plugin with `tutor enable distro` and run `tutor config save`

-  Check this:

- [ ] In the config.yml you have 
```yml
DOCKER_IMAGE_OPENEDX: docker.io/ednxops/distro-edunext-edxapp:nuez
DOCKER_IMAGE_OPENEDX_DEV: docker.io/ednxops/distro-edunext-edxapp-dev:nuez
EDX_PLATFORM_REPOSITORY: https://github.com/eduNEXT/edunext-platform.git
EDX_PLATFORM_VERSION: ednx-release/nuez.master
PLUGINS:
- distro
``` 
- [ ]  One you init and start your environment check you can use `eox-tenant version === 6.0.1`
